README.md

```diff
@@ -13,33 +13,35 @@ Authors
 Installation
 ------------
 
- 1. Add the PhoneNumberBundle to your dependencies:
-
-        // composer.json
-
-        {
-           // ...
-           "require": {
-               // ...
-               "misd/phone-number-bundle": "~1.0"
-           }
-        }
+ 1. Add the PhoneNumberBundle to your dependencies in the `composer.json`:
+
+```json
```

Comment:
Well, I am not the bundle maintainer. :) I find it more readable the way it is now, but the final decision is, of course, up to @thewilkybarkid.